docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
-    }, {
-        name: 'id',
-        type: 'string',
-        default: '',
-        description: 'An identifier.',
-        allowedTypes: '',
-    }, {
-        name: 'selectable',
-        type: 'bool',
-        default: 'false',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: '',
-    }, {
-        name: 'singleLine',
-        type: 'bool',
-        default: '',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: 'medium, small',
-    }, {
-        name: 'size',
-        type: 'enum',
-        default: 'small',
-        description: 'The size of a table\'s body of cells.',
-        allowedTypes: 'medium, small',
-    }, {
-        name: 'stackable',
-        type: 'bool',
-        default: 'false',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: '',
-    }, {
-        name: 'stretch',
-        type: 'bool or enum',
-        default: '',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: 'very',
-    }, {
-        name: 'striped',
-        type: 'bool',
-        default: 'false',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: '',
-    }, {
-        name: 'style',
-        type: 'object',
-        default: '',
-        description: 'Headers may be formatted with a subheader',
-        allowedTypes: '',
-    },
-];
-
-export default function CollectionsTable() {
+import TableSampleVerticalAlign from './tableSampleVerticalAlign';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as tableDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/dataDisplay/table';
+/* eslint-enable import/no-named-default, import/extensions */
+
+function DocsPersonPanel() {
+    const descriptionCopy = tableDoc.description;
+
     return (
         <Main page="headers">
-            <TitleBar title="Table" />
+            <TitleBar title="Person Panel" />
 
             <Main.Content>
-                <Card>
-                    <Header size="large">Props</Header>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {descriptionCopy}
+                    </Typography>
+
+                    <Heading
+                        anchorLink="example"
+                        variant="h2"
+                    >
+                        Table
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A standard table.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSample').default}
+                >
+                    <TableSample />
+                </Example>
+
```

Comment:
We removed the "Basic Table" example!  Was that on purpose?

Reply:
Yes. The plan is one day to have basic always be `true`... if you will. No need for the other table's styling.

Reply:
Fixed.

Reply:
Understood.  Resolving this thread.